app/frontend/src/features/profile/edit/EditHostingPreference.tsx

```diff
@@ -160,319 +158,321 @@ export default function HostingPreferenceForm() {
-            }}
-          />
-          <Controller
-            control={control}
-            defaultValue={user.smokingAllowed}
-            name="smokingAllowed"
-            render={({ onChange }) => (
-              <Autocomplete
-                disableClearable={false}
-                defaultValue={user.smokingAllowed}
-                forcePopupIcon
-                freeSolo={false}
-                getOptionLabel={(option) => smokingLocationLabels[option]}
-                multiple={false}
-                options={[
-                  SmokingLocation.SMOKING_LOCATION_YES,
-                  SmokingLocation.SMOKING_LOCATION_WINDOW,
-                  SmokingLocation.SMOKING_LOCATION_OUTSIDE,
-                  SmokingLocation.SMOKING_LOCATION_NO,
-                ]}
-                onChange={(e, value) =>
-                  onChange(value ?? SmokingLocation.SMOKING_LOCATION_UNKNOWN)
-                }
-                renderInput={(params) => (
-                  <ProfileTextInput
-                    {...params}
-                    label={ACCEPT_SMOKING}
-                    name="smokingAllowed"
-                    className={classes.field}
-                  />
-                )}
-              />
-            )}
-          />
-          <ProfileMarkdownInput
-            id="aboutPlace"
-            label={ABOUT_HOME}
-            name="aboutPlace"
-            defaultValue={user.aboutPlace}
-            control={control}
-            className={classes.field}
-          />
-          <Controller
-            control={control}
-            defaultValue={user.sleepingArrangement}
-            name="sleepingArrangement"
-            render={({ onChange }) => (
-              <Autocomplete
-                disableClearable={false}
-                defaultValue={user.sleepingArrangement}
-                forcePopupIcon
-                freeSolo={false}
-                getOptionLabel={(option) => sleepingArrangementLabels[option]}
-                multiple={false}
-                options={[
-                  SleepingArrangement.SLEEPING_ARRANGEMENT_PRIVATE,
-                  SleepingArrangement.SLEEPING_ARRANGEMENT_COMMON,
-                  SleepingArrangement.SLEEPING_ARRANGEMENT_SHARED_ROOM,
-                  SleepingArrangement.SLEEPING_ARRANGEMENT_SHARED_SPACE,
-                ]}
-                onChange={(e, value) =>
-                  onChange(
-                    value ??
-                      SleepingArrangement.SLEEPING_ARRANGEMENT_UNSPECIFIED
-                  )
-                }
-                renderInput={(params) => (
-                  <ProfileTextInput
-                    {...params}
-                    label={SPACE}
-                    name="sleepingArrangement"
-                    className={classes.field}
-                  />
-                )}
-              />
-            )}
-          />
-          <div className={classes.checkboxContainer}>
-            <div>
+        <>
```

Comment:
Is it just me or would this diff be way smaller without this (it's not needed if there's only one child)